style.css

```diff
@@ -128,6 +231,30 @@ body {
   font: 700 66px/87px PTSerif, arial, helvetica, sans-serif;
 }
 
+@media (max-width: 1366px) {
+  .main__title {
+    margin: 0 0 0 -108px;
+    top: 16px;
+    left: 275px;
+    font: 700 50px/66px PTSerif, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width : 1024px) {
+  .main__title {
+    margin: 0 0 0 -62px;
+    left: 70px;
+    font: 700 50px/66px PTSerif, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 768px) {
+  .main__title {
+    margin: 24px 35px 32px -92px;
+    font: 700 30px/40px PTSerif,arial,helvetica,sans-serif;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6362272)